clients/vscode/src/chat/chatPanel.ts

```diff
@@ -25,7 +25,7 @@ export function createThreadFromWebview<Self = Record<string, never>, Target = R
   );
 }
 
-export function createClient(webview: WebviewView, api: ClientApi): ServerApi {
+export function createClient(webview: WebviewView | WebviewPanel, api: ClientApi): ServerApi {
```

Comment:
```suggestion
export function createClient(webview: Webview, api: ClientApi): ServerApi {
```

Reply:
I think we can do this refactor in a separate PR to make sure the refactor of `ChatSideViewProvider` as lean as possible. @wsxiaoys